python/mspasspy/algorithms/resample.py

```diff
@@ -229,6 +229,21 @@ def resample(self, mspass_object):
             )
             mspass_object.set_npts(n_resampled)
             mspass_object.dt = self.dt
+            # Check for "sampling_rate" attribute
+            if mspass_object.is_defined("sampling_rate"):
+                sampling_rate = mspass_object["sampling_rate"]
+                # Check if sampling_rate is consistent with 1/dt
+                if abs(sampling_rate - 1.0 / mspass_object.dt) > 1e-6:
+                    # Record inconsistency in error log (elog)
+                    message = "sampling_rate inconsistent with 1/dt; updating to 1/dt"
+                    mspass_object.elog.log_error("TimeSeries2Trace",
+                                                 message,
+                                                 ErrorSeverity.Invalid)
```

Comment:
The `ErrorSeverity` should be `Complaint` instead of `Invalid`. Also, the algorithm name here is wrong. This is inside `resample` not `TimeSeries2Trace`. There are similar issues to the rest of the changes here.

Reply:
I fixed it and have pushed it.